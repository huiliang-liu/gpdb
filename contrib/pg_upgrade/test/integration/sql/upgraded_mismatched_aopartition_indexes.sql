SELECT * FROM mismatched_partition_indexes;
