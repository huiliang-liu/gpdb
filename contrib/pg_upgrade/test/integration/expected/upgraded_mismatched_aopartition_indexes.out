SELECT * FROM mismatched_partition_indexes;
 a | b     | c 
---+-------+---
 2 | boss  | 2 
 1 | apple | 1 
(2 rows)
